ValveResourceFormat/IO/MapExtract.cs

```diff
@@ -822,9 +824,42 @@ private static ContentFile GeneratePhysicsTagMaterial(string surfaceName, IFileL
         materialContentFile.FileName = HammerMeshBuilder.proceduralPhysMaterialsPath + surfaceName + ".vmat";
         assetReferences.Add(materialContentFile.FileName);
 
+        GeneratePhysicsTagTexture(surfaceName, "D:/S2 tools");
```

Comment:
todo: generate them into the exported folder (with additional content files or whatever)